### DISCUSSION TITLE
Is there some sort of special thing I need to do to get seperate html pages to work other than my index.html?

### DISCUSSION THREAD

**Discussion by gchis66** [vercel/community#5574] [Help]
### Question

I have a completely vanilla js/html/css project that I have multiple pages for my terms of service and privacy policy setup for and everything works fine in my local environment but when i push the changes to github and they take affect here, the other pages no longer work and instead when I click to navigate to them I get the following:
404: NOT_FOUNDCode:

NOT_FOUND
ID:

iad1::nk9zz-1705992936100-175826c2b12a

the html files are all in github though and have been uploaded successfully, not sure why im getting a not found on vercel

Probably something obvious, but any help is appreciated

**Comment by amyegan** [comment]
Hey @gchis66. Do the pages show correctly if you include the `.html` file extension, as in `example.com/about.html`? If that's the case, you can enable [`cleanUrls`](https://vercel.com/docs/projects/project-configuration#cleanurls) in a [`vercel.json`](https://vercel.com/docs/projects/project-configuration#) file. That will let visitors see the correct page even when they leave off the file extension.

**Reply by gchis66** [reply]
Hi @amyegan I actually do have the file extension .html included in the links and still receive the 404 error. Not sure what the issue is but I have all of these files in the root of the project if that matters.

**Reply by amyegan** [reply]
I was able to get an example working without running into any 404 errors ([repo](https://github.com/amyegan/html-vercel) & [example site](https://html-vercel-three.vercel.app/)). I imagine there is something more complex than this in your site setup though.

If you share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), someone here may be able to recognize what's going wrong.

**Reply by gchis66** [reply]
Thanks for making the effort to respond. Could you check out my repo? https://github.com/gchis66/palm-reader-ai   I think im missing something very obvious lol
